### DISCUSSION TITLE
Repositories of Expansions

### DISCUSSION THREAD

**Discussion by Andre601** [PlaceholderAPI/PlaceholderAPI#510] [(1) Announcements]
PlaceholderAPI uses expansions to provide support for all the different placeholders.
This brings the unwanted side-effect that many errors are caused by said expansions and not PlaceholderAPI.

To keep the issue-tracker of this Repository clean and up-to-date are we not accepting issues regarding problems with expansions.

To help you find, where you may ask for support about an expansion can you find a list of links for the expansion below.

> *Not seeing your expansion here?*
> *Comment below with a link to your Expansion's GitHub Repository to get it added here.*

## PlaceholderAPI
These expansions can be found on the [PlaceholderAPI Organisation](https://github.com/PlaceholderAPI).

- AcidIsland [[`PlaceholderAPI/AcidIsland-Expansion`][acidisland]]
- Animations [[`PlaceholderAPI/Animations-Expansion`][animations]]
- Armor [[`PlaceholderAPI/Armor-Expansion`][armor]]
- ASkyBlock [[`PlaceholderAPI/ASkyBlock-Expansion`][askyblock]]
- AutoRank [[`PlaceholderAPI/AutoRank-Expansion`][autorank]]
- BungeeCord [[`PlaceholderAPI/BungeeCord-Expansion`][bungeecord]]
- ChatReaction [[`PlaceholderAPI/ChatReaction-Expansion`][chatreaction]]
- CheckItem [[`PlaceholderAPI/CheckItem-Expansion`][checkitem]]
- Config [[`PlaceholderAPI/Config-Expansion`][config]]
- Custom [[`PlaceholderAPI/Custom-Expansion`][custom]]
- Daddy [[`PlaceholderAPI/Daddy-Expansion`][daddy]]
- DeluxeTags [[`PlaceholderAPI/DeluxeTags-Expansion`][deluxetags]]
- Enjin [[`PlaceholderAPI/Enjin-Expansion`][enjin]]
- Essentials [[`PlaceholderAPI/Essentials-Expansion`][essentials]]
- EzRanksPro [[`PlaceholderAPI/EzRanksPro-Expansion`][ezrankspro]]
- Geolocation [[`PlaceholderAPI/Geolocation-Expansion`][geolocation]]
- Gradient [[`PlaceholderAPI/Gradient-Expansion`][gradient]]
- Heroes [[`PlaceholderAPI/Heroes-Expansion`][heroes]]
- InvFreeze [[`PlaceholderAPI/InvFreeze-Expansion`][invfreeze]]
- Javascript [[`PlaceholderAPI/Javascript-Expansion`][javascript]]
- KotlinScript [[`PlaceholderAPI/KotlinScript-Expansion`][kotlinscript]]
- LemonMobCouns [[`PlaceholderAPI/LemonMobCoins-Expansion`][lemonmobcoins]]
- MegaEconomy [[`PlaceholderAPI/MegaEconomy-Expansion`][megaeconomy]]
- MultiEconomy [[`PlaceholderAPI/MultiEconomy-Expansion`][multieconomy]]
- Parties [[`PlaceholderAPI/Parties-Expansion`][parties]]
- Pinger [[`PlaceholderAPI/Pinger-Expansion`][pinger]]
- Player [[`PlaceholderAPI/Player-Expansion`][player]]
- PlayerTime [[`PlaceholderAPI/PlayerTime-Expansion`][playertime]]
- PluginInfo [[`PlaceholderAPI/PluginInfo-Expansion`][plugininfo]]
- RedisBungee [[`PlaceholderAPI/RedisBungee-Expansion`][redisbungee]]
- RelCon [[`PlaceholderAPI/RelCon-Expansion`][relcon]]
- ScoreboardObjectives [[`PlaceholderAPI/Expansion-ScoreboardObjectives`][scoreboardobjectives]]
- Server [[`PlaceholderAPI/Server-Expansion`][server]]
- SmashOrPass [[`PlaceholderAPI/SmashOrPass-Expansion`][smashorpass]]
- Sound [[`PlaceholderAPI/Sound-Expansion`][sound]]
- Statistics [[`PlaceholderAPI/Statistics-Expansion`][statistics]]
- Statz [[`PlaceholderAPI/Statz-Expansion`][statz]]
- SuperbVote [[`PlaceholderAPI/SuperbVote-Expansion`][superbvote]]
- Thirst [[`PlaceholderAPI/Thirst-Expansion`][thirst]]
- Towny [[`PlaceholderAPI/Towny-Expansion`][towny]]
- TownyChat [[`PlaceholderAPI/TownyChat-Expansion`][townychat]]
- uSkyBlock [[`PlaceholderAPI/uSkyBlock-Expansion`][uskyblock]]
- Vault [[`PlaceholderAPI/Vault-Expansion`][vault]]
- WorldGuard [[`PlaceholderAPI/WorldGuard-Expansion`][worldguard]]

## 3rd-Party
These are expansions which aren't within the PlaceholderAPI Organisation on GitHub, but have their own repository.

- Formatter [[`Andre601/Formatter-Expansion`][formatter]]
- Math [[`Andre601/Math-Expansion`][math]]
- RandomColor [[`Andre601/RandomColor`][randomcolor]]
- Shortcut [[`Andre601/Shortcut-Expansion`][shortcut]]
- RNG [[`Kqliber/Expansion-RNG`][rng]]

[acidisland]: https://github.com/PlaceholderAPI/AcidIsland-Expansion
[animations]: https://github.com/PlaceholderAPI/Animations-Expansion
[armor]: https://github.com/PlaceholderAPI/Armor-Expansion
[askyblock]: https://github.com/PlaceholderAPI/ASkyBlock-Expansion
[autorank]: https://github.com/PlaceholderAPI/AutoRank-Expansion
[bungeecord]: https://github.com/PlaceholderAPI/BungeeCord-Expansion
[chatreaction]: https://github.com/PlaceholderAPI/ChatReaction-Expansion
[checkitem]: https://github.com/PlaceholderAPI/CheckItem-Expansion
[config]: https://github.com/PlaceholderAPI/Config-Expansion
[custom]: https://github.com/PlaceholderAPI/Custom-Expansion
[daddy]: https://github.com/PlaceholderAPI/Daddy-Expansion
[deluxetags]: https://github.com/PlaceholderAPI/DeluxeTags-Expansion
[enjin]: https://github.com/PlaceholderAPI/Enjin-Expansion
[essentials]: https://github.com/PlaceholderAPI/Essentials-Expansion
[ezrankspro]: https://github.com/PlaceholderAPI/EzRanksPro-Expansion
[geolocation]: https://github.com/PlaceholderAPI/Geolocation-Expansion
[gradient]: https://github.com/PlaceholderAPI/Gradient-Expansion
[heroes]: https://github.com/PlaceholderAPI/Heroes-Expansion
[invfreeze]: https://github.com/PlaceholderAPI/InvFreeze-Expansion
[javascript]: https://github.com/PlaceholderAPI/Javascript-Expansion
[kotlinscript]: https://github.com/PlaceholderAPI/KotlinScript-Expansion
[lemonmobcoins]: https://github.com/PlaceholderAPI/LemonMobCoins-Expansion
[megaeconomy]: https://github.com/PlaceholderAPI/MegaEconomy-Expansion
[multieconomy]: https://github.com/PlaceholderAPI/MultiEconomy-Expansion
[parties]: https://github.com/PlaceholderAPI/Parties-Expansion
[pinger]: https://github.com/PlaceholderAPI/Pinger-Expansion
[player]: https://github.com/PlaceholderAPI/Player-Expansion
[playertime]: https://github.com/PlaceholderAPI/PlayerTime-Expansion
[plugininfo]: https://github.com/PlaceholderAPI/PluginInfo-Expansion
[redisbungee]: https://github.com/PlaceholderAPI/RedisBungee-Expansion
[relcon]: https://github.com/PlaceholderAPI/RelCon-Expansion
[scoreboardobjectives]: https://github.com/PlaceholderAPI/Expansion-ScoreboardObjectives
[server]: https://github.com/PlaceholderAPI/Server-Expansion
[smashorpass]: https://github.com/PlaceholderAPI/SmashOrPass-Expansion
[sound]: https://github.com/PlaceholderAPI/Sound-Expansion
[statistics]: https://github.com/PlaceholderAPI/Statistics-Expansion
[statz]: https://github.com/PlaceholderAPI/Statz-Expansion
[superbvote]: https://github.com/PlaceholderAPI/SuperbVote-Expansion
[thirst]: https://github.com/PlaceholderAPI/Thirst-Expansion
[towny]: https://github.com/PlaceholderAPI/Towny-Expansion
[townychat]: https://github.com/PlaceholderAPI/TownyChat-Expansion
[uskyblock]: https://github.com/PlaceholderAPI/uSkyBlock-Expansion
[vault]: https://github.com/PlaceholderAPI/Vault-Expansion
[worldguard]: https://github.com/PlaceholderAPI/WorldGuard-Expansion

[formatter]: https://codeberg.org/Andre601/Formatter-Expansion
[math]: https://codeberg.org/Andre601/Math-Expansion
[randomcolor]: https://github.com/Andre601/RandomColor
[shortcut]: https://github.com/Andre601/Shortcut-Expansion
[rng]: https://github.com/Kqliber/Expansion-RNG

**Comment by Clexus** [comment]
Please add Sound Expansion
I have a bug to report

**Reply by Andre601** [reply]
No idea what the sound expansion actually is, so I can't add it.

**Reply by Clexus** [reply]
> No idea what the sound expansion actually is, so I can't add it.

[THIS](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#sound)

**Reply by Andre601** [reply]
Funnycube created a repository for it now.
You can find it here: https://github.com/PlaceholderAPI/Sound-Expansion

**Comment by SerMonPiteInc** [comment]
PlaceholderAPI (Statistical) does not work. I'm trying to make time in the game, but when I entered the necessary placeholders, the digit '0' stands at the place of time and time does not go. What to do? I'm from Russia
Here is a screenshot that time is not going - https://ibb.co/FnT3CVK

Here is a photo from the plugin config for board - https://ibb.co/Npb7Xjk

**Reply by Andre601** [reply]
As mentioned in your issue, report this to the **issue tracker of the expansion**.

In your case: https://github.com/PlaceholderAPI/Statistics-Expansion

**Comment by Clexus** [comment]
please add [NumberFormatter](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#numberformatter) Expansion, I didn't see any explanation of the usage of it, even on its original [site](https://polymart.org/resource/papi-numberformatter.3316)

**Reply by Andre601** [reply]
This discussion only lists expansions which have a GitHub or other Code Repository.

**Comment by Clexus** [comment]
Add [World](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#world) Expansion
